oak_runtime/src/lib.rs

```diff
@@ -154,18 +154,18 @@ struct NodeStopper {
 
 impl NodeStopper {
     /// Sends a notification to the Node and joins its thread.
-    fn stop_node(self) -> thread::Result<()> {
-        let node_name = &self.node_name;
+    fn stop_node(self, node_id: NodeId) -> thread::Result<()> {
+        let node_debug_id = NodeInfo::construct_debug_id(&self.node_name, node_id);
```

Comment:
optional: Perhaps add a `get_debug_id` function to `NodeStopper` to be more consistent with `NodeInfo` and `Channel`. With this change, it probably makes sense to change `NodeInfo::construct_debug_id` into a normal function outside `NodeInfo`.

Reply:
Added a `get_debug_id` to `NodeStopper`. :) How would you go about moving `construct_debug_id` to its own function? 

I suppose I could share it across `Channel`s and `Node`s via:

```
enum Id {
    NodeId,
    ChannelId,
}

/// Returns a unique debug_id consisting out of the provided name and [`NodeId`].
fn construct_debug_id(name: &str, id: Id) -> String {
    // ...
}
```

Reply:
I am not too worried about `Channel`. But yeah, `construct_debug_id` does not have to be inside `NodeInfo`. I don't think the signature needs to be changed.